go.mod

```diff
@@ -1,45 +1,57 @@
 module github.com/dezswap/cosmwasm-etl
 
-go 1.20
+go 1.21
 
 require (
-	github.com/CosmWasm/wasmd v0.27.0
+	cosmossdk.io/math v1.4.0
+	cosmossdk.io/x/evidence v0.1.1
+	cosmossdk.io/x/feegrant v0.1.1
+	cosmossdk.io/x/upgrade v0.1.4
+	github.com/CosmWasm/wasmd v0.53.4
 	github.com/DATA-DOG/go-sqlmock v1.5.0
 	github.com/aws/aws-sdk-go-v2 v1.16.7
 	github.com/aws/aws-sdk-go-v2/config v1.15.13
 	github.com/aws/aws-sdk-go-v2/credentials v1.12.8
 	github.com/aws/aws-sdk-go-v2/service/s3 v1.27.1
 	github.com/aws/smithy-go v1.12.0
 	github.com/bxcodec/faker/v3 v3.8.0
-	github.com/cosmos/cosmos-sdk v0.45.6
-	github.com/davecgh/go-spew v1.1.1
+	github.com/cometbft/cometbft v0.38.12
+	github.com/cosmos/cosmos-sdk v0.50.14
```

Comment:
Currently cube is targeting release v1.8.
It use cosmos-sdk v0.53.4
https://github.com/xpladev/xpla/blob/release/v1.8.x/go.mod
All the major changes were made in 50, so I think there won't be any major problems if only a few dependencies like wasmd and ibc-go change. 🙇‍♂️